src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/ExtendedCourtCaseRequest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@AllArgsConstructor
+public class ExtendedCourtCaseRequest {
+    private final String caseNo;
+    private final String courtCode;
+    private final String courtRoom;
+    private final List<HearingDay> hearingDays;
+    private final List<OffenceRequest> offences;
```

Comment:
In the new feed, the offences are a property of the defendant. They're not a direct part of the case, although we could model them this way with a reference to the defendant. 
None of this is really crucial because we are learning I suppose.